fuzz/decode_receive.cc

```diff
@@ -0,0 +1,91 @@
+#include <picotls.h>
+#include <stdio.h>
+#include <netdb.h>
+#include "quicly.h"
+#include "quicly/defaults.h"
+#include "quicly/frame.h"
+#include "../deps/picotls/include/picotls/openssl.h"
+
+void __sanitizer_cov_trace_pc(void)
+{
+}
+
+int save_session_ticket_cb(ptls_save_ticket_t *_self, ptls_t *tls, ptls_iovec_t src)
+{
+    return 0; 
+}
+
+static int on_client_hello_cb(ptls_on_client_hello_t *_self, ptls_t *tls, ptls_on_client_hello_parameters_t *params)
+{
+    return 0;
+}
+
+static ptls_key_exchange_algorithm_t *key_exchanges[128];
+static ptls_cipher_suite_t *cipher_suites[128];
+
+static ptls_save_ticket_t save_session_ticket = {save_session_ticket_cb};
+static ptls_on_client_hello_t on_client_hello = {on_client_hello_cb};
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
+{
+
+	ptls_context_t tlsctx = {.random_bytes = ptls_openssl_random_bytes,
+                                .get_time = &ptls_get_time,
+                                .key_exchanges = key_exchanges,
+                                .cipher_suites = ptls_openssl_cipher_suites,
+                                .require_dhe_on_psk = 1 , 
+                                .save_ticket = &save_session_ticket,
+                                .on_client_hello = &on_client_hello};
+
+
+	int ret;
+	quicly_context_t ctx;
+	ctx = quicly_spec_context;
+	ctx.tls = &tlsctx;
+
+	quicly_decoded_packet_t p;
+
+	struct sockaddr sa;
+	socklen_t salen;
+		
+	quicly_conn_t *conn = NULL;
+	quicly_cid_plaintext_t next_cid;
+	const char* host = "127.0.0.1";
+	const char* port = "4422";
+	ptls_iovec_t *resumption_token = (ptls_iovec_t *)malloc(sizeof(ptls_iovec_t));
```

Comment:
I do not think we need a pointer (with allocate memory), as the only use of the value in relation to other functions is passing the value of the pointer in `quicly_connect`. This can be `ptls_iovec_t resumption_token = ptls_iovec_init(NULL, 0)`.